roles/splunk_standalone/molecule/snowflake/tests/test_snowflake.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python
+#
+# Test more esoteric options configurable using Ansible variables
+from __future__ import absolute_import
+import os
+
+import testinfra.utils.ansible_runner
+
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('all')
+
+SPLUNK_HOME = "/opt/splunk"
+SPLUNK_EXEC = "{}/bin/splunk".format(SPLUNK_HOME)
+SPLUNK_USER = SPLUNK_GROUP = "splunk"
+
+
+def test_splunk_user_group(host):
+    user = host.user(SPLUNK_USER)
+    assert user.name == SPLUNK_USER
+    assert user.group == SPLUNK_GROUP
+
+def test_splunk_installation(host):
+    d = host.file(SPLUNK_HOME)
+    assert d.is_directory
+    assert d.user == SPLUNK_USER
+    assert d.group == SPLUNK_GROUP
+    f = host.file(SPLUNK_EXEC)
+    assert f.is_file
+    assert f.user == SPLUNK_USER
+    assert f.group == SPLUNK_GROUP
+
+def test_splunk_running(host):
+    output = host.run("{} status".format(SPLUNK_EXEC))
+    assert "running" in output.stdout
+
+def test_user_seed(host):
+    f = host.file("{}/etc/system/local/user-seed.conf".format(SPLUNK_HOME))
+    assert not f.exists
+
+def test_ui_login(host):
+    f = host.file("{}/etc/.ui_login".format(SPLUNK_HOME))
+    assert f.exists
+    assert f.user == SPLUNK_USER
+    assert f.group == SPLUNK_GROUP
+
+def test_splunk_version(host):
+    f = host.file("{}/etc/splunk.version".format(SPLUNK_HOME))
+    assert f.exists
+    assert f.user == SPLUNK_USER
+    assert f.group == SPLUNK_GROUP
+    assert f.contains("VERSION=8.0.3")
+
+def test_splunk_pid(host):
+    f = host.file("{}/var/run/splunk/splunkd.pid".format(SPLUNK_HOME))
+    assert f.exists
+    assert f.user == SPLUNK_USER
+    assert f.group == SPLUNK_GROUP
+
+def test_mongod_lock(host):
+    f = host.file("{}/var/lib/splunk/kvstore/mongo/mongod.lock".format(SPLUNK_HOME))
+    assert f.exists
+    assert f.user == SPLUNK_USER
+    assert f.group == SPLUNK_GROUP
+
+def test_bin_splunk(host):
+    f = host.file("{}".format(SPLUNK_EXEC))
+    assert f.exists
+    assert f.user == SPLUNK_USER
+    assert f.group == SPLUNK_GROUP
+
+def test_splunk_hec_inputs(host):
+    f = host.file("{}/etc/apps/splunk_httpinput/local/inputs.conf".format(SPLUNK_HOME))
+    assert f.exists
+    assert f.user == SPLUNK_USER
+    assert f.group == SPLUNK_GROUP
+    assert f.contains("[http]")
+    assert f.contains("disabled = 0")
+    assert f.contains("[http://splunk_hec_token]")
+    assert f.contains("token = qwerty789")
+
+def test_inputs_conf(host):
+    f = host.file("{}/etc/system/local/inputs.conf".format(SPLUNK_HOME))
+    assert f.exists
+    assert f.user == SPLUNK_USER
+    assert f.group == SPLUNK_GROUP
+    assert f.contains("[splunktcp://9999]")
+    assert f.contains("disabled = 0")
+
+def test_splunk_ports(host):
```

Comment:
Are we intending to test the different default ports - basically are we anticipating that these will ever change or that we'll want to use this for testing custom ports?

Reply:
Yeah, I'm purposely testing the ability to set different ports via the different options in the variables here: roles/splunk_standalone/molecule/snowflake/molecule.yml